### ISSUE TITLE
Python isinstance does not work with imported Java classes

### ISSUE THREAD

**Issue by mike-hunhoff** [ninia/jep#438]
e.g.

```python
>>> from ghidra.program.database.data import *
...
>>> type(dt)
<class 'ghidra.program.database.data.FunctionDefinitionDB'>
>>> print(FunctionDefinitionDB)
class ghidra.program.database.data.FunctionDefinitionDB
>>> type(FunctionDefinitionDB)
<class 'jep.PyJClass'>
>>> isinstance(dt, FunctionDefinitionDB)
Traceback (most recent call last):
  File "/Users/pieceofsummer/.ghidra/.ghidra_10.1.5_PUBLIC/Extensions/Ghidrathon-1.0.0/data/python/jepeval.py", line 66, in jepeval
    more_input_needed = _jepeval(line)
  File "/Users/pieceofsummer/.ghidra/.ghidra_10.1.5_PUBLIC/Extensions/Ghidrathon-1.0.0/data/python/jepeval.py", line 49, in _jepeval
    exec(compile(line, "<string>", "single"), globals(), globals())
  File "<string>", line 1, in <module>
TypeError: isinstance() arg 2 must be a type, a tuple of types, or a union
```

I see a workaround based on jep code found [here](https://github.com/ninia/jep/blob/b926ca4af1df7a496ea0a8cb7d030bbe310de898/src/test/python/test_import.py#L58-L59) that requires users to pass `__pytype__` e.g. `FunctionDefinitionDB.__pytype__`. Is there a way to use `isinstance` without a workaround?

See https://github.com/mandiant/Ghidrathon/issues/13 for more details.

**Comment by bsteffensmeier** [comment]
[PEP-3119](https://peps.python.org/pep-3119/) makes it easy to override this behavior. I have submitted #440 with a proposed fix for this. Unfortunately since we just released 4.1 it will be awhile before this makes it into a release.

**Comment by mike-hunhoff** [comment]
#440 looks great @bsteffensmeier . Would love to see this in a point release to help improve Jep's usability but understand you just made a release. In the mean time I think we can implement a workaround but we will keep our eyes open for the next Jep release, thank you!

**Comment by ndjensen** [comment]
Fixed in 4.1.1.
